pallets/block-rewards/src/tests.rs

```diff
@@ -5,7 +5,7 @@ use sp_runtime::traits::BadOrigin;
 use super::*;
 use crate::mock::*;
 
-const REWARD: u128 = 100;
+const REWARD: u128 = 100 * crate::mock::ExistentialDeposit::get();
```

Comment:
Maybe it should be an addition instead?

Reply:
It doesn't really matter, as long as it's more than the ED and the genesis balance is sufficient. I made it 100 * ED since in some tests we set the total reward to 2 or 10 * REWARD or claim twice, so 100+ would work.

Reply:
That weird. It works for me using `+` there. Maybe it was another error that is already fixed?

I really think having a `+` there improves understanding of what's going on in the tests.

Reply:
Maybe, I don't know. And I don't share the feeling that + improves the understanding over, both operations basically set the reward to a value higher than the ED itself. We should just leave a comment saying that the Reward should be bigger than the ED and maybe make the reward some more reasonable value like 1 CFG for example.

Reply:
Sorry if I am not understanding your point of view, but I really don't follow what we are discussing here. 

> you are setting more total rewards than you need

Why do I need to set it to a bare minimum? Does it even matter?

> With `+`, you set exactly the amount used in the tests.

Isn't this the amount used in the tests? Isn't that exactly the purpose of this constant? 🤔 what am I missing here? Keep in mind I've gone through hundreds of failing tests so I might be forgetting some detail here.

> Actually, multiplying Balance by itself is an operation that should not "exist". EUR * EUR does not exist.

Really don't follow what the discussion is here. By setting the mock test reward value to 100 times the ED, we are not "multiplying Balance by itself", we are multiplying ED (a value of type Balance) by a magnitude of 100. 100 is the multiplier in this case, but it's not quite that relevant; what's relevant is that `REWARD` needs to be a value higher than ED, and `100 * ED` accomplishes that just fine, as well as `100 +`, `99 +`, `42 *` or whatever other option would as long as there's no overflow :) As I mentioned above, we could also set the reward to 1 CFG; as long as it's > ED and the accounts used in the tests have sufficient balance, it all serves the purpose of covering what these tests need to cover.

Reply:
Yeah, I understand that any value over some limit will fulfill the test's requirement. My point is that setting the exact value you need to fulfill those requirements helps the reader to know exactly what's going on in the tests.

If I read `100 + ED`, I know somehow that `100` will be handled in the tests, and the `ED` is the part required to fulfill the existential deposit requirement.

Reading `100 * ED` seems like that amount will be distributed to 100 accounts or something. Now, `100` is no longer an amount but a multiplier of `ED`, which leads the reader to think this.

Reply:
Anyways, just a Nit point 😄 . I'll revert this change because it was added along with other changes in the fixes PR I've pushed over this

Reply:
There was no need to revert the change, I just wanted to understand your point of view :) I honestly don't share the feeling you share but that's ok. I also think that keeping the value just right above the minimum it could be not only is not mapping the reality in production but it also makes the tests more brittle.

In any case, feel free to set this back to your `100 +`, specially if some other peers shares your feeling here 🙏

Reply:
I personally favor the `+`.

Reply:
Fixed and discussion closed in https://github.com/centrifuge/centrifuge-chain/pull/1577/commits/1ba6cddd527a57e86f03ffe3954f13ef9ddacf63 where I've also added a comment. Thanks for your input 👍